core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -298,11 +298,11 @@ public String toString() {
               + "\"Owners\" one.")
           .build();
   public static final PropertyKey UNDERFS_HDFS_CONFIGURATION =
-      new Builder(Name.UNDERFS_HDFS_CONFIGURATION)
-          .setDefaultValue(String.format(
-              "${%s}/core-site.xml:${%s}/hdfs-site.xml", Name.CONF_DIR, Name.CONF_DIR))
-          .setDescription("Location of the hdfs configuration file.")
-          .build();
+          new Builder(Name.UNDERFS_HDFS_CONFIGURATION)
```

Comment:
@lucasliunju the indentation is not correct according to the style guide of this codebase, can you revert the indentation?